configs/permissions/complicance.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We should probably address the typo in the file name. complicance.json -> compliance.

Reply:
Nice catch 👍